gnovm/cmd/gno/test.go

```diff
@@ -69,14 +115,14 @@ func (c *testCfg) RegisterFlags(fs *flag.FlagSet) {
 		&c.updateGoldenTests,
 		"update-golden-tests",
 		false,
-		"writes actual as wanted in test comments",
+		`writes actual as wanted for "Output:" and "Realm:" instructions`,
```

Comment:
```suggestion
		`writes actual as wanted for "Output:", "Error:" and "Realm:" instructions`,
```

Reply:
Seems strange. Could you investigate further and let us know if there's a reason, or consider consistency across updatable comments?

Reply:
I don't know the reason, but I realized these instructions took inspiration from the [yaegi repo][0], probably you already had noticed that :)

If you want to push for consistency, I suggest to create a separate issue because I don't think this PR is a good place for such change. WDYT?

[0]: https://github.com/traefik/yaegi/blob/8a6061cc86ed7ef4d232dd9a65593275952e08fe/interp/interp_file_test.go#L87

Reply:
Just summoning @mvertes before deciding, because it may be a super quick fix. 

Anyway if you prefer, feel free to keep it for another PR.

Reply:
I really prefer an other PR, since this one is *docs scoped*.

-> Issue created https://github.com/gnolang/gno/issues/1015